src/VSTS-Bot.Api/Dialogs/ConnectDialog.cs

```diff
@@ -356,7 +356,7 @@ private static string GeneratePin()
         [OnSerializing]
         private void OnSerializingMethod(StreamingContext context)
         {
-            this.vstsService = GlobalConfiguration.Configuration.DependencyResolver.GetService<IVstsService>();
```

Comment:
What is the reason this is disabled?
When you are in the connect flow, the connectdialog gets serialized and put on the stack.
When it is deserialized, you have lost the reference to the vsts service.

Reply:
We don't need these methods because:
1. They being called during serialization, not during **de**serialization, so that is useless. 
2. _this.vstsService_ being initialized through the parametrized constructor call during deserialization.